client/src/pages/ListPaymentOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import CreditCardIcon from '@material-ui/icons/CreditCard';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import AccountBalanceIcon from '@material-ui/icons/AccountBalance';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+
+function ListPaymentOptions(props) {
+
+    return (
+        <div>
+            <ListItem button onClick={e => props.setStripePage("profile")}>
```

Comment:
i think you might want to wrap your `<ListItem>`s around a `<List>` here instead of a div
https://material-ui.com/components/lists/#simple-list